pex/util.py

```diff
@@ -9,74 +9,34 @@
 import tempfile
 from hashlib import sha1
 from site import makepath  # type: ignore[attr-defined]
+from zipfile import ZipFile
 
 from pex.common import atomic_directory, safe_mkdir, safe_mkdtemp
 from pex.compatibility import (  # type: ignore[attr-defined]  # `exec_function` is defined dynamically
     PY2,
     exec_function,
 )
-from pex.third_party.pkg_resources import (
-    find_distributions,
-    resource_isdir,
-    resource_listdir,
-    resource_string,
-)
+from pex.third_party.pkg_resources import Distribution, find_distributions
 from pex.tracer import TRACER
+from pex.typing import TYPE_CHECKING
 
+if TYPE_CHECKING:
+    if PY2:
+        from hashlib import _hash as _Hash
+    else:
+        from hashlib import _Hash
+    from typing import Any, Callable, IO, Iterable, Iterator, Optional
 
-class DistributionHelper(object):
-    @classmethod
-    def access_zipped_assets(cls, static_module_name, static_path, dir_location=None):
```

Comment:
Unused, beyond a test.

Reply:
Ah, this old thing. It is used but we don't have API markers yet. This commit's ba58a854 review is lost inside Twitter but was used to allow a Pexed up application that knew about Pexing to extract its assets; i.e: client code called (calls) it. This shouldn't be killed until API / deprecations / etc is set up. Perhaps add a note to this effect since you did the right thing given all the discoverable information.

Reply:
Added back. Thanks.